packages/common/src/services/auth/hedgehog.ts

```diff
@@ -34,6 +41,18 @@ export const createHedgehog = ({
   localStorage,
   createKey = getPlatformCreateKey()
 }: HedgehogConfig): HedgehogInstance => {
+  const getAuthHeaders = async (wallet: EthWallet) => {
```

Comment:
nit: this code is also duplicated in `identityService.ts` in `_getSignatureHeaders`.. not sure if there's a way to dedup. probably too many interdependencies.

Reply:
Yeah I know. It was kind of intentional. This class has to be initialized before identityService is, so I duplicated the things from that class which are needed to make the small number of requests to identity in this class.
I could move it into a shared utility in the same folder?